packages/wallets/coin98/src/adapter.ts

```diff
@@ -36,11 +37,10 @@ export interface Coin98WalletAdapterConfig {}
+    icon = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACIAAAAiCAYAAAA6RwvCAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAABmJLR0QA/wD/AP+gvaeTAAAAB3RJTUUH5gIcAzknxQ5v8AAABqtJREFUWMO9l9tvVNcVxn/rXOY+Y3vs8d3EBhubhAAhBAJCIWkJDpBKrVBbHEQfWtqXtlLbv6AP/QfSqg9V1UZRaFGlKJEChIAILQRCkBsSOYbGxhh1HGKMPZ6L5z5zzu7DjGOMZzBpa9bL2Tr7sr7z7W99+2xZu3YtSimAJhE5AgyISDfgBBAR5p/3tr9O333PHHATkWMCfwSmRAQ9GAwC9AJ/KANpEhGDcixKhoD8z0AMICQizwMbgUEgogFNwKtAP6CV0iseQWjAHuC3QJMGHAFevHeEqopjRQDuBn6sAQMLTJRDqsAQWW7R/5aZAQ3oXtpXOeGKwCgtvEajXB2L31fZghWSjihxGhV71OLPtxWIAhEFShBRaFplfmxboRA0KS0iAprGA0OhMCp3lHAoBQ5D6Olw0rfKRcBrEE9ZfB7OcfN2HsuGedkopRARutqcPLXOR0OdSTxpMTSaZiycQ6mFsZWiIhCREoj6GoODL9Syfb2PgFdH0zUQSGRszn+a4s1zURIpu/zVQv/2Wgb21dMScmKYGgqYTVi8dXaWt8/OUixW31u9vr7+1wsAFiB7XTpH9gfZtdGPreD6v3MM38qSzSuagyaPd7nxunWGbmYoFBXb1vv42cEm6gImNyeyfHw9RS6vaG92sqHXy/RskbFwdonBzbcrMmIr2LbOw9Y+D3MZm7+cjXJpOI2mlSb1bwswsKee55/288/P01wdSdO/vYaAz+DyUJLf/22KSNyixqfzk+82s3dXHXufq+XSJ3Ok0nbFLaooI4chbO5x4zCED4dTnP14Dr9H45XdQXRNOH4xztWRNF63xtN9Xmr9Bh1NDvJ5mzOX40xFCggQTVic+iBGKm3R1uigodaoapZLgKgykGDAwLJhZCJH0YJvbPbTv83Pzg1esnnFaDiLUtAYNDD0ebrLVaPAshVKKexyZhF5gGNXEKsAloJCQSECfk8Ja3SuyJfTBeIpCxHweXREIF9QJJIWwzczDF5P8eV0gdXtTvxeg0zWJj5X5M3TETrbXMzGi5UrR1WpmlzeZvxOjifXuNjxhJcr/0pzZnCOD6+lSaZtOlscbH3ci23DaDhLOmvz2jvT9HW5+dF3QvR2efC4dQqW4tbtHO9eiHHiH1GSaWuRSJetGqUgmbHZ1O1mVZODzmYHuUKJ+g3dbg6/VE93h4vwVIFjZ2aZS9vsfMrPzw8209vpBoRUxsbt0ujqcLFlvY940mLkVpb7c83vaVUfGZ/Mc+z9GD/oD7Kp280Tq93ki+BxazgcOnciBV4/OcMXdwusaXdyeF8DNX6dK58leftclDszBRqCJt/+ZpDnnqnh0LdCjIWzDI2kl7iyUNXQSgMvDCWJJCxefMZPT7sLt0sjnrYZmUhx+kqCsS/yiMCODT5aGkxuhLP87tgdpiJFAj6dyUiB8GSegM9g60Y/Lzxbw/CNzMOJ9d6wFQyNZxiZyFHn13E6NLJ5RTxlUygqdE0wTY3VbS5EYPBaSay7tgQ49HKIE+ejnDgf4+LVObY86Wd1uwu3UyOTsx8OiFIKp6nRXG8iwORskem4hWCBCB6XxmPNTmwFM7Eiur4gcqWgsc5kTYeTlpDjnvcKXRekygFYBQjU+XV+eSCE163x6lszfDaexdAFy1Zs6vHw0wON3I0V+c1rk9yeLgCwvtuD3xvj74NxHmtz8t7FGE5T2LTOi64Lk9N5shXYgCrOKgLxlE00aRGqMXh5ew2tDSamIXS1ONm/I0AwoDM1WyCRshgcThFPWWzq9XBoXwOmqfHG8WlyeZvv721g5+YAmazNpatzFKocfJU1IpDO2ZweTLCm1cnWPg8djQ5mEhatIZPWkIPpWJH3LiewbLg2nuHkBzG+t6eeA7uDPLvRz91okVCdSWe7C90QTl2I8dGnc2hfx0cANBEmI0XiKYv2RgetDSbtjQ7cLo3wVIHXT0X4ZDRTsnVVMrZsXtEactDa6GBVq5PagEk0UeSdc1GOHp8hnbG/Kt3FPgLS09OjKgFZaAvNQZPeVS5qfDrRpMXoRI6ZmLVkURGhrdFBb6eb2oBBMmMzFs4xMZkv/eVVuBN9lW85IPOHVfnHD0TQtBJjlS5YS8aWx1e7cC2rkfvFW0pMxUUWqX8JSywbIoIGZJd0qGqzV+xCkdWA0UeXr3IopW5owFGg+JBTVgKHBRzVgD8D794H8VESchL4kwFEgF8ANrAfMB/Ex/9x1wplEL8CIoaUEtwCfggcBl4BWUvpKrqYmgcgeUiQAuSAUaX4qwhvAFGA/wAjUJfreOScGwAAADhlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAAqACAAQAAAABAAABLKADAAQAAAABAAABLAAAAAD7qKDdAAAAJXRFWHRkYXRlOmNyZWF0ZQAyMDIyLTAyLTI4VDAzOjU3OjMzKzAwOjAwYHFggAAAACV0RVh0ZGF0ZTptb2RpZnkAMjAyMi0wMi0yOFQwMzo1NzozMyswMDowMBEs2DwAAAASdEVYdGV4aWY6RXhpZk9mZnNldAAyNlMbomUAAAAYdEVYdGV4aWY6UGl4ZWxYRGltZW5zaW9uADMwMEW2lAMAAAAYdEVYdGV4aWY6UGl4ZWxZRGltZW5zaW9uADMwMNi5dXUAAAAASUVORK5CYII=';
```

Comment:
Any chance you have an original SVG instead of a PNG we can use? It should be much smaller.